<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppSettingsWidget</class>
 <widget class="QWidget" name="AppSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>73</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="iconButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/app</normaloff>:/icons/app</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>256</width>
               <height>256</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="nameEdit">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="placeholderText">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="packageNameLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>package.name</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QToolButton" name="iconResetButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/reset</normaloff>:/icons/reset</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="iconBrowseButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/browse</normaloff>:/icons/browse</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="iconInternalButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/app</normaloff>:/icons/app</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>16</number>
           </property>
           <item>
            <widget class="QCheckBox" name="userAppButton">
             <property name="text">
              <string>User Application</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="hideButton">
             <property name="text">
              <string>Visible</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="quickAccessButton">
             <property name="text">
              <string>Quick Access</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="uninstallButton">
             <property name="text">
              <string> Uninstall</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/appIcons/resources/icons/remove.svg</normaloff>:/appIcons/resources/icons/remove.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="applyButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/apply</normaloff>:/icons/apply</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="iconSelectorPage">
      <layout class="QVBoxLayout" name="iconSelectorLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QToolButton" name="closeIconsButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearCmdButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="searchButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/search</normaloff>:/icons/search</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
